main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsFeedInfoSchema.java

```diff
@@ -38,16 +40,21 @@ public interface GtfsFeedInfoSchema extends GtfsEntity {
 
   Locale defaultLang();
 
+  @Recommended
   @EndRange(field = "feed_end_date", allowEqual = true)
   GtfsDate feedStartDate();
 
+  @Recommended
   GtfsDate feedEndDate();
 
+  @Recommended
   String feedVersion();
 
+  @Recommended
   @FieldType(FieldTypeEnum.EMAIL)
   String feedContactEmail();
 
+  @Recommended
   @FieldType(FieldTypeEnum.URL)
   String feedContactUrl();
```

Comment:
According to the [spec](https://gtfs.org/schedule/reference/#feed_infotxt), this fields are optional, not recommended. I understand though that the file is recommended. @isabelle-dr What is your input on this? Should fields included in a recommended file be recommended although the spec says they are optional? I think they should stay optional, but I would be happy to hear other thoughts on this.

Reply:
This is related to #885.

Reply:
Perfect, thank you!